src/Compilers/CSharp/Portable/Symbols/SubstitutedParameterSymbol.cs

```diff
@@ -45,24 +45,21 @@ public override TypeSymbolWithAnnotations Type
             get
             {
                 var mapOrType = _mapOrType;
-                var type = mapOrType as TypeSymbolWithAnnotations;
-                if ((object)type != null)
+                if (mapOrType is TypeSymbolWithAnnotations)
```

Comment:
>if (mapOrType is TypeSymbolWithAnnotations) [](start = 16, length = 43)

nit: `if (mapOrType is TypeSymbolWithAnnotations typeSymbolWithAnnotations) return typeSymbolWithAnnotations;`